guest/toolbox/client.go

```diff
@@ -346,6 +347,8 @@ func (c *Client) Upload(ctx context.Context, src io.Reader, dst string, p soap.U
 		return err
 	}
 
+	p.Close = true // disable Keep-Alive connection to ESX
```

Comment:
Hmm, in the Envoy sidecar scenario in https://github.com/vmware/govmomi/pull/3227/files#diff-85b38e4cec5f84a008e29777420bd336a111a4685b883f55c4a7de9e96572a64R113 , I used DisableKeepAlives, will this supersede that?

Reply:
```	// DisableKeepAlives, if true, disables HTTP keep-alives and
	// will only use the connection to the server for a single
	// HTTP request.
```

```
	// Close indicates whether to close the connection after
	// replying to this request (for servers) or after sending this
	// request and reading its response (for clients).
	//
	// For server requests, the HTTP server handles this automatically
	// and this field is not needed by Handlers.
	//
	// For client requests, setting this field prevents re-use of
	// TCP connections between requests to the same hosts, as if
	// Transport.DisableKeepAlives were set.
```

Not quite sure I see the distinction..

Reply:
Ah, nvm. Scrolled up to see the actual structs, one's on the Request, one's on the Transport

Reply:
`transport.DisableKeepAlives = true` applies to every request made by the http.Client(s) using that transport. `Request.Close` only applies to that specific request.